spynnaker/pyNN/models/common/neuron_recorder.py

```diff
@@ -666,34 +667,44 @@ def _get_fixed_sdram_usage(self, vertex_slice):
         fixed_sdram += self.N_BYTES_PER_INDEX * total_neurons
         return fixed_sdram
 
-    def get_static_sdram_usage(self, vertex_slice):
+    def get_data_spec_sdram_usage(self, vertex_slice):
         n_record = len(self.__sampling_rates)
-        sdram = (
-            recording_utilities.get_recording_header_size(n_record) +
-            recording_utilities.get_recording_data_constant_size(n_record) +
-            self.get_sdram_usage_in_bytes(vertex_slice))
+        header_size = recording_utilities.get_recording_header_size(n_record)
+        # Size for n_vars, n_bitfields and _get_data
+        sdram_size = self.get_sdram_usage_in_bytes(vertex_slice)
+        sdram = (header_size + sdram_size)
         return int(sdram)
 
-    def get_variable_sdram_usage(self, vertex_slice):
-        fixed_sdram = 0
-        per_timestep_sdram = 0
+    def get_recording_sdram_usage(self, vertex_slice):
+        """
+        Find the SDRAM cost for recording this slice
+
+        :param vertex_slice: Slice being costed
+        :return: SDRAM cost
+        rtype: MultiRegionSDRAM
+        """
+        costs = MultiRegionSDRAM()
         for variable in self.__sampling_rates:
             rate = self.__sampling_rates[variable]
-            fixed_sdram += self._get_fixed_sdram_usage(vertex_slice)
```

Comment:
This is where the method now marked as dead was called; thus it is this change that makes the code dead!